src/dotnet/devcontainer-feature.json

```diff
@@ -1,35 +1,31 @@
 {
     "id": "dotnet",
-    "version": "1.1.4",
+    "version": "2.0.0",
     "name": "Dotnet CLI",
     "documentationURL": "https://github.com/devcontainers/features/tree/main/src/dotnet",
-    "description": "Installs the .NET CLI. Provides option of installing sdk or runtime, and option of versions to install. Uses latest version of .NET sdk as defaults to install.",
+    "description": "Installs .NET and its CLI. Offers options to install multiple SDK versions.",
     "options": {
         "version": {
             "type": "string",
             "proposals": [
                 "latest",
-                "7",
-                "6",
-                "3.1"
+                "lts",
+                "8.0",
+                "7.0",
+                "6.0"
             ],
             "default": "latest",
-            "description": "Select or enter a dotnet CLI version. (Available versions may vary by Linux distribution.)"
+            "description": "Version of .NET SDK to install. Use 'latest' for the latest version, 'lts' for the latest LTS version, 'X.Y' for a specific release, or 'X.Y.Z' for a specific version."
         },
-        "runtimeOnly": {
-            "type": "boolean",
-            "default": false,
-            "description": "Install just the dotnet runtime if true, and sdk if false."
-        },
-        "installUsingApt": {
-            "type": "boolean",
-            "default": true,
-            "description": "If true, it installs using apt instead of the release URL"
+        "additionalVersions": {
+            "type": "string",
+            "default": "",
+            "description": "Comma-separated list of additional .NET SDK versions to install. Use 'latest' for the latest version, 'lts' for the latest LTS version, 'X.Y' for a specific release, or 'X.Y.Z' for a specific version."
         }
     },
     "containerEnv": {
-        "DOTNET_ROOT": "/usr/local/dotnet/current",
-        "PATH": "/usr/local/dotnet/current:${PATH}"
+        "DOTNET_ROOT": "/usr/share/dotnet",
```

Comment:
This change comes from a want to run a GitHub action inside a prebuilt Dev Container (https://docs.github.com/en/actions/using-jobs/running-jobs-in-a-container)

I noticed that the `setup-dotnet` GitHub action installs to `/usr/share/dotnet` by default
https://github.com/actions/setup-dotnet#environment-variables

If you want to use `setup-dotnet` in a Dev Container that already has dotnet installed in `/usr/local`, then you have to override the `DOTNET_INSTALL_DIR` env var of the action. Otherwise the existing installation will not be detected.

I recommend aligning the installation directory of the .NET feature with the default installation directory of the `setup-dotnet` action so they work together without requiring configuration of the `DOTNET_INSTALL_DIR` env var.

Reply:
Hmm, thanks for the explanation. The logic makes sense to me. 
I'll need to dig around for the actual reason of preferring this PATH. (@Chuxel wondering if you might know!) 

Note: We'd need to update https://github.com/devcontainers/images/blob/main/src/universal/.devcontainer/local-features/setup-user/install.sh#L73-L78 post this PR is merged.

Reply:
If this is too problematic, I think it is also fine to keep the old directory and add `DOTNET_INSTALL_DIR` to the `containerEnv`.

``` json
"containerEnv": {
    "DOTNET_ROOT": "/usr/local/dotnet/current",
    "DOTNET_INSTALL_DIR": "$DOTNET_ROOT",
    "PATH": "$PATH:$DOTNET_ROOT:~/.dotnet/tools"
}
```

This might be the best way to ensure maximum compatibility, although I find the `current` directory a bit weird and unnecessary beyond maintaining compatibility.

Reply:
Also check out the docs for DOTNET_ROOT

> Specifies the location of the .NET runtimes, if they are not installed in the default location. The default location on Windows is C:\Program Files\dotnet. The default location on macOS is /usr/local/share/dotnet. The default location on Linux varies depending on distro and installment method. **The default location on Ubuntu 22.04 is /usr/share/dotnet (when installed from packages.microsoft.com) or /usr/lib/dotnet (when installed from Jammy feed)**. 

https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-environment-variables#dotnet_root-dotnet_rootx86

Reply:
>  "DOTNET_ROOT": "/usr/share/dotnet",

Discussed this with the team, as `"/usr/share/dotnet"` syncs with GitHub Actions, and also with the [official dotnet image](https://hub.docker.com/_/microsoft-dotnet), it would be great to sync the PATH. Thanks for the change! 

As PATH changes can be highly breaking, we do need to signal it with major version bump. As this PR already does that, it seems fine. 👍